docs/content/mongocxx-v3/installation.md

```diff
@@ -180,7 +180,6 @@ an additional option, `/Zc:__cplusplus` to opt into the correct definition of `_
     -G "Visual Studio 15 2017 Win64"            \
     -DCMAKE_CXX_STANDARD=17                     \
     -DCMAKE_CXX_FLAGS="/Zc:__cplusplus"         \
-    -DBOOST_ROOT=C:\local\boost_1_59_0          \
```

Comment:
Not necessary. Boost is one of the polyfill options, it is not needed when compiling with a C++17 compiler.